src/figuresClasses.ts

```diff
@@ -1,19 +1,92 @@
 export interface Figure {
+  color: string;
+  shape: string;
```

Comment:
```suggestion
  shape: Shape;
```
create type for shape and color like this type Shape = 'triangle' | 'circle' | 'rectangle';